eth/tracers/api_test.go

```diff
@@ -889,7 +889,7 @@ func TestTraceChain(t *testing.T) {
 			t.Error("Missing tracing block")
 		}
 		if ref != rel {
-			t.Errorf("Ref and deref actions are not equal, ref %d rel %d", ref, rel)
+			t.Errorf("Ref and deref actions are not equal, ref %d rel %d", ref.Load(), rel.Load())
```

Comment:
Myeah, I think it is not right to do `ref != rel` on two atomic types, on the line above. That's why I suggested using `have, want := ... ` instead

Reply:
my fault! update it.